creator-node/src/services/prometheusMonitoring/prometheusMetrics.constants.js

```diff
@@ -0,0 +1,54 @@
+const promClient = require('prom-client')
+
+const MetricTypes = Object.freeze({
+  COUNTER: promClient.Counter,
+  GAUGE: promClient.Gauge,
+  HISTOGRAM: promClient.Histogram,
+  SUMMARY: promClient.Summary
+})
+
+const MetricNames = Object.freeze({
+  HTTP_REQUEST_DURATION_SECONDS_HISTOGRAM: 'http_request_duration_seconds',
+  STORAGE_PATH_SIZE_GAUGE: 'storage_path_size_UNIT_TODO_gauge',
+  STORAGE_PATH_USED_GAUGE: 'storage_path_used_UNIT_TODO_gauge',
```

Comment:
The unit should always be in a base unit, so `bytes` in this case.

Also, our Audius naming docs use `_current` at the moment for gauges, but we should match the official docs if that's incorrect.

Reply:
Let me think on this further. I know we chose a prefix to differentiate gauges, but I can't remember the use case.

Let me get back to you on this please.

Reply:
sg!

Reply:
I found an example!

http://grafana.audius.co/d/000000002/load-test?orgId=1&viewPanel=43&editPanel=43&from=now-24h&to=now-1m

```
audius_dn_celery_running_tasks
```

This is a bad metric name, even though the panel's tooltip/description provides the necessary information. (Notice the verbiage around the stale gauge data as it relates to our sister conversation within this PR. Granted, this does need to be fixed within the business logic and we have a ticket for that work.)

> Shows how long the longest running task (across all nodes) has been running. Currently, metrics will not reset a stat back to 0 upon completion, so you'll see horizontal lines indicating it hasn't run since the last metric was updated. Dips to 0 mean the task has started again. Dips to non-0 mean that while the current task has completed, the same task was running in parallel on another node, and the metric is showing the new currently running longest task.

Note: apologies this metric is named incorrectly, I was OOO for a bit after our off-site and didn't think through this too clearly upon my return.

In order to understand this metric fully, we need to understand Dheeraj's request to Johannes and myself:

Question 1: "I want to know how long the currently running tasks have been running."

Even though that's what Dheeraj requested, when presented with the panel Dheeraj didn't know what to make of the data and if I'm not mistaken, you were also confused about this panel.

First let's change the metric name to be nicer, something like:

```
audius_dn_celery_task_duration_seconds
```

With this properly named metric, I can easily see what you and Dheeraj expected from the metric which answers a different question:

Question 2: "I want to know how long it takes for celery tasks to complete."

However, this was not the original intention of the metric, this was:

Question 1: "I want to know how long the currently running tasks have been running."

Ideally, `audius_dn_celery_task_duration_seconds` would be a histogram (so that we can get statistical buckets), but let's say a gauge would work well here as well (especially since a gauge is a subset of a histogram and only a gauge is needed to very directly answer the Question 2). In this case, a histogram and/or gauge can collect the metric **after** it has completed its run and we can keep the metric name `audius_dn_celery_task_duration_seconds`. (Again, we wouldn't ideally want a "gauge" here since we can probably do some stats magic after the fact with the additional information that histograms provide.)

However, there is no way to see if a job is currently stuck since we will only report that a job took 1000 hours after it competes its job, if we don't restart the server due to the stuck job before it generates the metric.

In this case, if we were to answer both Questions 1 and 2, we would want two different metrics to answer how long the currently running tasks are taking and how long a completed task took, respectively:

```
audius_dn_celery_task_duration_seconds_current
audius_dn_celery_task_duration_seconds
```